pkg/interfaces/contracts/pool-hooks/IMevHook.sol

```diff
@@ -186,4 +210,37 @@ interface IMevHook {
      * @param newPoolMevTaxThreshold The new threshold to be set in a pool
      */
     function setPoolMevTaxThreshold(address pool, uint256 newPoolMevTaxThreshold) external;
+
+    /**
+     * @notice Checks if sender is MEV tax exempt.
+     * @dev A MEV tax exempt sender pays only the static swap fee percentage, regardless of the paid priority fee.
+     * @param sender Sender to check if is MEV tax exempt
+     * @return mevTaxExempt true if sender is MEV tax exempt
+     */
+    function isMevTaxExempt(address sender) external view returns (bool mevTaxExempt);
+
+    /**
+     * @notice Gets the number of MEV tax exempt senders.
+     * @return length Number of MEV tax exempt senders
+     */
+    function getMevTaxExemptSendersLength() external view returns (uint256 length);
+
+    /**
+     * @notice Gets the sender from list of MEV tax exempt senders at position `index`.
+     * @param index Index of sender
+     * @return sender Address of the MEV tax exempt sender at position `index`
+     */
+    function getMevTaxExemptSendersAt(uint256 index) external view returns (address sender);
```

Comment:
I don't have a super strong opinion, but I think a mapping is enough for this application and we don't really need a set to enumerate addresses in the allowlist.